src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsItemsListener.php

```diff
@@ -48,13 +48,13 @@ public function onCommentAfterCreate(Event $event)
 	 *
 	 * @since   1.0
 	 */
-	public function onCommentAfterUpdate(Event $event)
+	public function onItemAfterSubmit(Event $event)
 	{
 		// Pull the arguments array
 		$arguments = $event->getArguments();
 
```

Comment:
if that works here and we jump into the when a new item gets submited we can remove that hack and move it to this place?

https://github.com/joomla/jissues/blob/master/src/App/Tracker/Controller/Issue/Submit.php#L71-L78